starter-code/src/components/CountryList.js

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from 'react';
+import { CountryContext } from '../contexts/CountryContext'; 
+import { Link } from 'react-router-dom';
+
+const CountryList = () => {
+  const {countries} = useContext(CountryContext);
+
+  return(
+    <div className="col-5" style={{maxHeight: "90vh", overflow: "scroll"}}>
+      <div className="list-group">
+        {countries.map(country => {
```

Comment:
Igual aquí en el `map` podéis hacer el `destructuring` a country para no tener que escribir todo después.